### PR TITLE
Switches vscode dmi editor to new extension.

### PR DESCRIPTION
**Pull request by AnturK**
Shameless self-promotion.

image
image

### CODE REVIEW

File: .vscode/extensions.json [hunk 1]
```diff
@@ -6,6 +6,7 @@
 		"arcanis.vscode-zipfs",
 		"dbaeumer.vscode-eslint",
 		"stylemistake.auto-comment-blocks",
-		"Donkie.vscode-tgstation-test-adapter"
+		"Donkie.vscode-tgstation-test-adapter",
+		"anturk.dmi-editor"
```
Comment (tralezab) [inline .vscode/extensions.json:10]: WOOOOW

**Review by tralezab — APPROVED** [review]
(just kidding)

**Comment by ZeWaka** [comment]
https://github.com/goonstation/goonstation/commit/1a2fde145c67f6c3e25b2bc9440cacc89f65e966 2slow

**Review by ZeWaka — COMMENTED** [review]
also lmao what are these reccomendations, why do you still have "gbasood.byond-dm-language-support"

spacemandmm superseded the syntax highlighting many months ago

**Comment by ZephyrTFA** [comment]
Bro, you mean I don't have to open DME when I want to adjust a sprite anymore? This will revolutionary for my laziness

**Review by JohnFulpWillard — APPROVED** [review]

**Pull request merged**